plugins/woocommerce/includes/admin/meta-boxes/views/html-product-attribute-inner.php

```diff
@@ -104,5 +104,17 @@ class="multiselect attribute_values wc-taxonomy-term-search"
 			</div>
 		</td>
 	</tr>
+	<?php
+	if ( ! isset( $is_variations_screen ) ) {
```

Comment:
I think we should always do this action, regardless of screen.

Otherwise, any custom attribute settings would not be able to be set if added via the variations screen.

What do you think?

Reply:
I am being more conservative here because I can't predict whether it would work or not. This UI is reused on the variations screen in a very specific context. So I'd prefer keeping it as it is. 

Also, it seems we're going to remove the use of this UI in the variations screen soon, as the design was updated

Reply:
>  it seems we're going to remove the use of this UI in the variations screen soon, as the design was updated

Reply:
> I am being more conservative here because I can't predict whether it would work or not. This UI is reused on the variations screen in a very specific context. So I'd prefer keeping it as it is.